.buildkite/basic/browser-pipeline.yml

```diff
@@ -178,6 +178,7 @@ steps:
           - ios_10
         depends_on: "browser-maze-runner-bs"
         timeout_in_minutes: 30
+        soft_fail: true
```

Comment:
As discussed IRL this would unfortunately mean allow fails on safari 16 too.  Suggest bumping to ios_11 or remove use of the matrix and just have separate steps, with one set to soft_fail.